python/cuml/test/test_umap.py

```diff
@@ -268,3 +268,99 @@ def test_umap_fit_transform_against_fit_and_transform():
     fit_embedding_diff_input = cuml_model.transform(data[1:],
                                                     convert_dtype=True)
     assert joblib.hash(ft_embedding) != joblib.hash(fit_embedding_diff_input)
+
+
+@pytest.mark.parametrize('n_components', [2, 25])
```

Comment:
Commenting on this line because Github won't allow me to comment on lines that weren't changed. We should also explicitly set the random state parameter for `test_umap_transform_on_iris`, which is one of the pytest cases that initially prompted this feature. The averaging that's being done currently should also be removed.